src/user-guide/running-workflows/scheduler-log-files.rst

```diff
@@ -24,11 +24,50 @@ The information logged here includes:
 
    Workflow log files are primarily intended for human eyes. If you need
    to have an external system to monitor workflow events automatically, use:
-   
+
    * Event hooks (see :cylc:conf:`flow.cylc[scheduler][events]` and
      :cylc:conf:`flow.cylc[runtime][<namespace>][events]`).
    * The GraphQL interface (can be accessed via GraphiQL in the Cylc GUI).
    * The sqlite *workflow run database*
      (see :ref:`Workflow Run Databases`)
-     
+
    Rather than parse the log files.
+
+
+.. _scheduler Logs.Cylc message:
+
+Cylc Message
+^^^^^^^^^^^^
+
+.. warning::
+
+   Using Cylc Message to add information to the scheduler logs should
+   be used sparingly to avoid overwhealming users with output.
+
+   You can use severity "DEBUG" to log information only in debug mode.
+
+The Scheduler Log also records messages sent by ``cylc message`` allowing users
+to add custom messages to this log.
+
+For example, if your task contained the following code:
+
+.. code-block:: shell
+
+   some_function
+   cylc message -- "${CYLC_WORKFLOW_ID}" "${CYLC_TASK_JOB}" \
+       "ERROR:some_function failed."
+
+Your log should produce output similar to:
+
+.. code-block:: shell
+
+   ERROR - [21491012T0410Z/mytask running job:01 flows:1] (received)some_function failed ERROR at 2023-04-14T11:36:35+01:00
```

Comment:
```suggestion
.. code-block:: shell

   cylc message -- "ERROR:some_function failed."

Your log should produce output similar to:

.. code-block:: none

   ERROR - [21491012T0410Z/mytask running job:01 flows:1] (received)some_function failed ERROR at 2023-04-14T11:36:35+01:00
```

In task environments, Cylc automatically loads the context from environment variables, example here:

https://github.com/cylc/cylc-flow/blob/master/tests/functional/restart/53-task-prerequisites/flow.cylc